packages/libevent.rb

```diff
@@ -1,33 +1,24 @@
 require 'package'
 
 class Libevent < Package
-  description 'The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached.'
+  description 'Event notification library'
   homepage 'http://libevent.org/'
-  version '2.1.8-0'
-  source_url 'https://github.com/libevent/libevent/releases/download/release-2.1.8-stable/libevent-2.1.8-stable.tar.gz'
-  source_sha256 '965cc5a8bb46ce4199a47e9b2c9e1cae3b137e8356ffdad6d94d3b9069b71dc2'
+  version '2.1.11'
+  source_url 'https://github.com/libevent/libevent/archive/release-2.1.11-stable.tar.gz'
```

Comment:
You probably don't need `./autogen.sh` with the release download.

Reply:
When I do updates I always download and examine the release tarball. This one didn't have configure present so autogen.sh was needed.

Reply:
Did you try `https://github.com/libevent/libevent/releases/download/release-2.1.11-stable/libevent-2.1.11-stable.tar.gz`?